source/connecting/OIDC-to-OIDC.txt

```diff
@@ -0,0 +1,105 @@
+.. _c2c-conn-OIDC2OIDC:
+
+================================================
+Connect Two Clusters that Support OpenID Connect
+================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+.. include:: /includes/mongosync-and-OIDC.rst
+
+.. include:: /includes/fact-connect-intro
+
+This page provides instructions to connect clusters that support OpenID
+Connect (OIDC) using {+c2c-full-product-name+}.
+
+About this Task
+---------------
+
+- Both the source and destination clusters must use MongoDB 8.0 or 
```

Comment:
Mongosync doesn't support server 8.0 right now.

What server versions support OIDC? Is it only 7.0+?

Reply:
cc @pmeredit